metric_learn/sdml.py

```diff
@@ -81,7 +81,6 @@ def _fit(self, pairs, y):
     M, _, _, _, _, _ = quic(emp_cov, lam=self.sparsity_param,
                             msg=self.verbose,
                             Theta0=theta0, Sigma0=sigma0)
-    M = M + 1e-10 * np.eye(M.shape[0])  # to ensure M is positive semi-definite
```

Comment:
This is something I had added to pass tests but then I realized with the good initialization (see lines above, in the previous commit), I don't need this anymore so I delete it